rules/windows/process_creation/proc_creation_win_query_registry.yml

```diff
@@ -25,6 +25,8 @@ detection:
             - 'currentVersion\run' # Also covers the strings "RunOnce", "RunOnceEx" and "runServicesOnce"
             - 'currentVersion\policies\explorer\run'
             - 'currentcontrolset\services'
+            - 'hklm\sam'
+            - 'hklm\security'
```

Comment:
Query to these two shouldn't be bundled in the same rule, as the severity is higher than "low". Perhaps add a new rule?